packages/sdk/package-lock.json

```diff
@@ -0,0 +1,2842 @@
+{
+  "name": "casperlabs-sdk",
```

Comment:
Please delete this file, and use yarn to install new packages with `yarn add axios`